assests/javascript/index.js

```diff
@@ -44,6 +51,68 @@ async function addTask(task) {
   }
 }
 
+async function getAllTasks() {
+  try {
+    const response = await fetch(`${baseUrl}/todos`);
+    const {data} = await response.json();
+
+    return sortTasks(data);
+  } catch {
+    showConnecionErrorMassage();
+  }
+}
+
+async function showAllTasks() {
+  const tasks = await getAllTasks();
+
+  taskList.innerHTML = tasks.map((task) => {
+    return `<li class="task-box tooltip" id=${task.id}>
+              <input type="checkbox" class="task-check-box" ${isChecked(task.isDone)}></input>
+              <p class="task-title">${task.text}</p>
+              <i class="fa-solid fa-pen-to-square edit-task-icon"></i>
+              <i class="fa-solid fa-xmark delete-task-icon"></i>
+
+              <p class="tooltiptext">${convertTime(task.createdAt)}</p>
+            </li>`;}).join("");
+
+  filterTasks(tasks);
+}
+
+function convertTime(time) {
+  const dateTime = new Date(time);
+  const year = dateTime.toLocaleDateString("en-US", { year: "numeric" });
+  const month = dateTime.toLocaleDateString("en-US", { month: "short" });
+  const date = dateTime.toLocaleDateString("en-US", { day: "numeric" });
+
+  return `${date} ${month} ${year}`;
+}
+
+async function filterTasks(tasks) {
```

Comment:
Naming is not good. The function is not filtering anything. Think about this a bit more.

Reply:
replaced by `classifyTasks`